src/utils/jwtConfiguration.ts

```diff
@@ -0,0 +1,13 @@
+import jwt from 'jsonwebtoken'
+
+export const generateToken = (data: object) => {
+  const token = jwt.sign(data, process.env.JWT_SECRET || 'USER-AUTH', {
+    expiresIn: process.env.JWT_EXPIRE_TIME || 300000,
```

Comment:
```suggestion
    expiresIn: JWT_EXPIRE_TIME,
```
its better to import environment  from `src/config`